metron-interface/metron-config/src/app/service/sensor-indexing-config.service.spec.ts

```diff
@@ -101,12 +101,10 @@ describe('SensorIndexingConfigService', () => {
     });
 
     it('deleteSensorEnrichments', () => {
-      sensorIndexingConfigService.deleteSensorIndexingConfig('squid').subscribe(
-        result => {
-          expect(result.status).toEqual(200);
```

Comment:
I noticed this test because of a failing Type Error. However, because we are mocking the backend and injecting the response, this test is just checking what we are injecting and is unnecessary. As a result, I removed it.